src/main/java/binaries/melissa/BinaryArray.java

```diff
@@ -18,6 +18,13 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        int decimal = 0;
+		final int BASE_NUMBER = 2;
```

Comment:
Hello Melissa, thanks for the updates. We still need this indentation to be fixed, starting this line, tabs must be replaced with 4 spaces.